### PR TITLE
TF-581 Apply new icon for iOS/android/web

### PR DESCRIPTION
**Pull request by dab246**
### 1. Process
- Upgrade library in `jmap_dart_client` [https://github.com/linagora/jmap-dart-client/pull/29](https://github.com/linagora/jmap-dart-client/pull/29)

### 2. Result:

- Favicon Web
Screen Shot 2022-06-15 at 17 52 35

- Splash Screen Android 

<table>
<tr>
<th>Mobile</th>
<th>Tablet</th>
</tr>
<tr>
<td></td>
<td></td>
</tr>
</table>

- Splash Screen  iOS

<table>
<tr>
<th>iPhone</th>
<th>iPad</th>
</tr>
<tr>
<td></td>
<td></td>
</tr>
</table>

- AppIcon Mobile

<table>
<tr>
<th>Android</th>
<th>iOS</th>
</tr>
<tr>
<td></td>
<td></td>
</tr>
</table>

### CODE REVIEW

**Comment by hoangdat** [comment]
IMO, splash screen in Android is not scale correctly. use one PNG for it is not a good idea :)

File: contact/pubspec.yaml [hunk 1]
```diff
@@ -16,12 +16,12 @@ dependencies:
...
   equatable: 2.0.3
 
-  json_annotation: 4.4.0
+  json_annotation: 4.5.0
 
   jmap_dart_client:
     git:
-      url: https://github.com/linagora/jmap-dart-client.git
-      ref: master
+      url: https://github.com/dab246/jmap-dart-client.git
```
Comment (hoangdat) [inline contact/pubspec.yaml:23]: I merged it, please update it

**Comment by hoangdat** [comment]
1HD

**Comment by dab246** [comment]
> IMO, splash screen in Android is not scale correctly. use one PNG for it is not a good idea :)

I fixed it. Please see demo image.

**Comment by dab246** [comment]
> How about the splash screen in Tablet :)

Please see demo image.

**Review by hoangdat — APPROVED** [review]

**Review by ManhNTX — APPROVED** [review]

**Pull request merged**